connect_ext_ppr/service.py

```diff
@@ -40,3 +47,16 @@ def add_deployments(installation, listings, config, logger):
             db.expire_all()
             dep_ids = ', '.join([d.id for d in deployments])
             logger.info(f"The following Deployments have been created: {dep_ids}.")
+
+
+def validate_ppr_schema(dict_file: Dict[str, Any]):
+    try:
+        jsonschema.validate(dict_file, PPR_SCHEMA)
+    except jsonschema.ValidationError as ex:
+        error = ClientError(
```

Comment:
[SOL-C] Propose to make an error class to store all the errors in one place.
Otherwise we may get confused in the error codes. Do you agree?